scripts/exec.js

```diff
@@ -2,8 +2,8 @@
 //   Shell for UQCSBot (allows advanced use of commands)
 //
 // Commands:
-//   !`shell` _<command>_ - Executes the shell command
-//
+//   `!exec <COMMAND>` - Executes the shell command
```

Comment:
We really need to get @lewisjb to come up with a better description for this

Reply:
yeah I really didn't know what to do with it so just fixed the wrong word and decided it was for another day bahahahaha